pkg/drivers/hyperv/hyperv_windows.go

```diff
@@ -373,3 +376,34 @@ func (d *Driver) GetIP() (string, error) {
 
 	return resp[0], nil
 }
+
+func (d *Driver) GetSharedDirs() ([]drivers.SharedDir, error) {
+	dirs := []drivers.SharedDir{}
+	for _, dir := range d.SharedDirs {
+		if !smbShareExists(dir.Tag) {
+			return []drivers.SharedDir{}, drivers.ErrNotSupported
```

Comment:
Why "ErrNotSupported"? Could just return an empty list of shares if we don't have the share we expect?